spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnSelector.scala

```diff
@@ -2,9 +2,19 @@ package com.datastax.spark.connector
 
 import scala.language.implicitConversions
 
-sealed trait ColumnSelector
-case object AllColumns extends ColumnSelector
-case class SomeColumns(columns: NamedColumnRef*) extends ColumnSelector
+sealed trait ColumnSelector {
+  def toAliasMap: Map[String, String]
```

Comment:
I'd call it `aliasMap`. I don't think alias map preserves all the information that is the ColumnSelector